lk2nd/device/dts/msm8952/rules.mk

```
@@ -16,3 +16,6 @@ ADTBS += \
 	$(LOCAL_DIR)/msm8940-xiaomi-santoni.dtb \
 	$(LOCAL_DIR)/msm8956-mtp.dtb \
 	$(LOCAL_DIR)/msm8976-qrd.dtb \
+
+DTBS += \
+    $(LOCAL_DIR)/msm8917-huawei-agassi.dtb
```

```suggestion
    $(LOCAL_DIR)/msm8917-huawei-agassi.dtb \

```

let's always escape newlines so if we need to add anything it would not change old lines